compiler/semdata.nim

```diff
@@ -64,6 +64,7 @@ type
     efWantStmt, efAllowStmt, efDetermineType, efExplain,
     efAllowDestructor, efWantValue, efOperand, efNoSemCheck,
     efNoProcvarCheck, efNoEvaluateGeneric, efInCall, efFromHlo,
+    efInConcept
```

Comment:
You don't need this. 

You can detect if the compiler is currently evaluating a body of a concept by checking `c.matchedConcept != nil`